app/controllers/flights_controller.rb

```diff
@@ -1,5 +1,28 @@
 class FlightsController < ApplicationController
   def index
-    @flights = Flight.all
+    return @flights unless flights.empty?
+
+    render_empty_list_message
+  end
+
+  private
+
+  def flights
+    @flights ||= Flight.search_departure_flights_for(params)
+  end
+
+  def render_flights_list
+    render turbo_stream: turbo_stream.update(
+      'flights-index-list'.to_sym,
+      partial: 'partials/flights/list',
+      locals: { flights: }
+    )
+  end
```

Comment:
Is this method being used somewhere? I don't see it

Reply:
done!